store/store_asserts.go

```diff
@@ -109,6 +129,14 @@ func (s *Store) downloadAssertions(u *url.URL, decodeBody func(io.Reader) error,
 						return e
 					}
 				}
+				// default error handling
+
+				// is it v2 error?
+				if svcErr.Status == 0 && len(svcErr.ErrorList) > 0 {
+					return fmt.Errorf("assertion service error: [%s] %q", svcErr.ErrorList[0].Code, svcErr.ErrorList[0].Message)
```

Comment:
the v2 errors contain the title in the message, so we'll get:

"[not-found] not found: ..."

we should either just use Message and not code?  or strip title from message and putting into "[...]"